test/testsuite-full.js

```diff
@@ -1,4 +1,5 @@
 require('./testsuite-schema-wrapper')
 require('./testsuite-apollo-plugin')
+require('./testsuite-apollo4-plugin')
 require('./testsuite-envelop-plugin')
-require('./testsuite-validation-rule-express-graphql')
+// require('./testsuite-validation-rule-express-graphql') // deprecated, may be removed later or rewritent to another server if available
```

Comment:
Can this be left in and marked as deprecated in the readme?

Reply:
See comment for `"express-graphql"` dependency